Bukkit/pom.xml

```diff
@@ -114,6 +114,15 @@
                                     </includes>
                                 </filter>
                             </filters>
+                            <artifactSet>
+                                <includes>
+                                    <include>me.egg82:trollcommandsplusplus-common</include>
+                                    <include>org.bstats:bstats-bukkit</include>
+                                    <include>ninja.egg82:*</include>
+                                    <include>info.ronjenkins:slf4bukkit</include>
+                                    <include>org.slf4j:slf4j-api</include>
+                                </includes>
+                            </artifactSet>
```

Comment:
Better to exclude than include, since the worst-case scenario for include is random runtime errors and the worst-case for exclude is the main namespace is polluted and maybe a conflict happens. Either way, exclude is much easier to see just by opening the jar in 7-zip.